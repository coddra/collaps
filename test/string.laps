"\x48".
"\ttab\n".
"\u01af".
"\U0001f603".
."very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string"
