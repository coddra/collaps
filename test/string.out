H
	tab

Æ¯
ðŸ˜ƒ
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
very long string very long string very long string very long string
